Text/SplitImage/SplitImage.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <AzureFunctionsVersion>v2</AzureFunctionsVersion>
+    <RootNamespace>AzureCognitiveSearch.PowerSkills.Text.SplitImage</RootNamespace>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Magick.NET-Q16-AnyCPU" Version="7.15.0" />
+    <PackageReference Include="Microsoft.NET.Sdk.Functions" Version="1.0.28" />
+    <PackageReference Include="SixLabors.ImageSharp" Version="1.0.0-beta0007" />
+    <PackageReference Include="System.Drawing.Common" Version="4.7.0" />
+    <PackageReference Include="TiffLibrary.ImageSharpAdapter" Version="0.5.134-beta" />
+  </ItemGroup>
```

Comment:
@bertrand what do you think of adding this reference?

Reply:
added a disclaimer at the least..

---
In reply to: [368185057](https://github.com/Azure-Samples/azure-search-power-skills/pull/20#discussion_r368185057) [](ancestors = 368185057)